<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ChgTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_PrintTxt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ClearLable">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Label</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
